src/styles/index.ts

```diff
@@ -446,8 +452,7 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.lineHeightHero,
         },
 
-        textNewKansasNormal: {
-            fontFamily: FontUtils.fontFamily.platform.EXP_NEW_KANSAS_MEDIUM,
+        fontSizeNormal: {
```

Comment:
Is this a new style?

Reply:
Oh. There was a similar style named textNormal:
```
        textNormal: {
            fontSize: variables.fontSizeNormal,
        },
```
So I removed ```fontSizeNormal``` and use ```textNormal``` instead.